README.md

```diff
@@ -13,14 +13,25 @@ The tempest snap can be installed directly from the snap store:
 ## Quickstart
 To run Tempest, please follow the steps in [Tempest QuickStart](https://docs.openstack.org/tempest/latest/overview.html#quickstart) documentation.
 
-The tempest snap provides a set of pre-defined test lists with different focuses. For quickly verifying the behavior of the objective cloud and making sure tempest is correctly set up, you can run tests listed in the `readonly-quick.txt` file:
 
-    tempest run --load-list '@BUILTIN_TESTLISTS/readonly-quick.txt' 
+## Test Lists
+The tempest snap provides a set of pre-defined test lists with different profiles and a mechanism to specify them easily.
 
 `@BUILTIN_TESTLISTS` keyword will be automatically replaced by the full path of the directory storing the built-in test lists.
 
-**Note**: the single-quotation marks around the test list file path is required if `@BUILTIN_TESTLISTS` keyword is used (in another words, when running tempest with a built-in test list file). Fail to do that will result in wrong path error.
+Note that the single-quotation marks around the test list file path is required if `@BUILTIN_TESTLISTS` keyword is used (in another words, when running tempest with a built-in test list file). Fail to do that will result in wrong path error.
 
+### readonly-quick
+
+For quickly verifying the behavior of the objective cloud and making sure tempest is correctly set up
+
+    tempest run --load-list '@BUILTIN_TESTLISTS/readonly-quick'
+
+### refstack-2022.11
+
+Version 2022.11 of the [RefStack](https://refstack.openstack.org/#/) guidelines. The list includes the required and advisory tests of all platforms.
```

Comment:
Can you move links to bottom of the file to my changes in #7. Thanks

Reply:
And I also used fix length of line for README. Thanks

Reply:
fixed in the update.

Reply:
Only lines in `refstack-2022.11` section has been fixed. I suggest fixing the length for all changes you made in README because comparing to Robert's PR, some content was rearranged.